php/Terminus/Models/Environment.php

```diff
@@ -779,4 +780,50 @@ private function elementAsDatabase($element) {
     return $element;
   }
 
+  /**
+   * Load site info
+   *
+   * @param [string] $key Set to retrieve a specific attribute as named
+   * @return [array] $info
+   */
+  public function info($key = null) {
+    $path = sprintf('environments/%s', $this->get('id'));
+    $result = \TerminusCommand::request(
+      'sites',
+      $this->site->get('id'),
+      $path,
+      'GET'
+    );
+    $info = array(
+      'id'                    => $this->get('id'),
+      'watchers'              => null,
+      'diffstat'              => null,
+      'on_server_development' => null,
+      'dns_zone'              => null,
+      'randseed'              => null,
+      'target_ref'            => null,
+      'lock'                  => null,
+      'target_commit'         => null,
+      'styx_cluster'          => null,
+    );
```

Comment:
> PHP version info is linked to the site's info.

A site has a default php version, but it's actually set on an environment-by-environment basis.

> Why should we do the absolute minimum when all the data is easier to get than the minimum of data?

It's easier to promise interface backwards compatibility if we keep that interface as small and focused as possible. If there isn't an explicit reason/purpose for us to provide a data value, I'd rather not do it so we don't have to go to extra effort in the future to maintain it should it change.